cpp/src/ray/util/process_helper.cc

```diff
@@ -1,9 +1,93 @@
 #include "process_helper.h"
+#include "hiredis/hiredis.h"
+#include "ray/core.h"
+#include "ray/util/process.h"
+#include "ray/util/util.h"
 
 namespace ray {
 namespace api {
+std::shared_ptr<ProcessHelper> ProcessHelper::ProcessHelper_ = nullptr;
 
-void ProcessHelper::RayStart() { return; }
+static std::string GetSessionDir(std::string redis_ip, int port, std::string password) {
+  redisContext *context = redisConnect(redis_ip.c_str(), port);
```

Comment:
We should improve the code, use safe_hiredis will make the code shorter and safer, here is the example:
https://github.com/alipay/ray/blob/cpp_worker_safe_hiredis/src/ray/util/safe_hiredis_test.cc#L29